app/app/Vote.php

```diff
@@ -67,6 +67,11 @@ public function getSubtitleAttribute()
         }
     }
 
+    public function getDateAttribute()
```

Comment:
```suggestion
    public function getFormattedDateAttribute()
```

Und lass mal vielleicht lieber ein natürlicheres Datumsformat nehmen, irgendwie "Thursday, March 23rd" oder so anstelle von ISO